dashboard/app/models/concerns/pd/jot_form_backed_form.rb

```diff
@@ -82,7 +82,7 @@ def sync_from_jotform(form_id = nil)
           answers = submission[:answers]
 
           # Process answers here as a validation. An error will be raised if they don't match the questions.
-          processed_answers = questions_for_form(form_id).process_answers(answers)
+          processed_answers = questions_for_form(form_id).process_answers(answers, show_hidden_questions: true)
```

Comment:
Suggest adding a comment explaining why we write hidden values here

Reply:
Actually while writing that comment, I just realized this is redundant. It was needed for `skip_submission?` but also internally we process with hidden questions for `map_answers_to_attributes` below, so these can be combined and leverage the memoization. I'll remove this...